src/main/java/frc/robot/Robot.java

```diff
@@ -46,16 +43,20 @@ public void robotInit() {
       Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
       Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
       new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
-    } else if (IS_REPLAY) {
-      setUseTiming(false); // Run as fast as possible
-
-      String logPath =
-          LogFileUtil
-              .findReplayLog(); // Pull the replay log from AdvantageScope (or prompt the user)
-      Logger.setReplaySource(new WPILOGReader(logPath)); // Read replay log
-      Logger.addDataReceiver(
-          new WPILOGWriter(
-              LogFileUtil.addPathSuffix(logPath, "_sim"))); // Save outputs to a new log
+    } else {
+      try {
+        String logPath = LogFileUtil.findReplayLog();
+        if (logPath != null) {
+          setUseTiming(false); // Run as fast as possible
+
+          Logger.setReplaySource(new WPILOGReader(logPath)); // Read replay log
+          Logger.addDataReceiver(
+              new WPILOGWriter(
+                  LogFileUtil.addPathSuffix(logPath, "_sim"))); // Save outputs to a new log
+        }
+      } catch (Exception StringIndexOutOfBoundsException) {
```

Comment:
```suggestion
      } catch (StringIndexOutOfBoundsException e) {
```
Should be a specific exception class you want to catch.